app/controllers/items_controller.rb

```diff
@@ -17,9 +17,7 @@ def index
     @pagy, @items = pagy(item_scope)
   end
 
-  def show
-    @item = Item.listed_publicly.find(params[:id])
-  end
+  def show; end
```

Comment:
I think this change will disable displaying the item on the public inventory page.

Reply:
We added it as a method [below]( https://github.com/rubyforgood/circulate/pull/223/files#diff-2eb2bf110a48bee6c79d3bbb377a549eR41) in that controller.